scripts/conan/recipes/celeborn-cpp-client/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2025 ByteDance Ltd. and/or its affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <iostream>
+#include "celeborn/conf/CelebornConf.h"
+#include "celeborn/client/writer/PushState.h"
+
+using namespace celeborn;
+using namespace celeborn::client;
+
+int main() {
+  conf::CelebornConf conf;
+  conf.registerProperty(
+      conf::CelebornConf::kClientPushLimitInFlightTimeoutMs, std::to_string(0));
+  conf.registerProperty(
+      conf::CelebornConf::kClientPushLimitInFlightSleepDeltaMs,
+      std::to_string(0));
+  conf.registerProperty(
+      conf::CelebornConf::kClientPushMaxReqsInFlightTotal, std::to_string(0));
+  conf.registerProperty(
+      conf::CelebornConf::kClientPushMaxReqsInFlightPerWorker,
+      std::to_string(0));
+
+  std::unique_ptr<PushState> pushState = std::make_unique<PushState>(conf);
```

Comment:
Is this code enough to properly test the celeborn dependency? Should it try to construct a client/server?